monorepo-builder.php

```diff
@@ -100,63 +100,11 @@
             'cycle/migrations'          => '^1.0.1',
             'cycle/proxy-factory'       => '^1.2',
             'cycle/schema-builder'      => '^1.1',
-            'symplify/monorepo-builder' => '^8.3',
+            'symplify/monorepo-builder' => '^10.0',
             'vimeo/psalm'               => '^4.3',
         ],
     ]);
 
-    $parameters->set(Option::DIRECTORIES_TO_REPOSITORIES, [
-        // Bridge
-        'src/Bridge/DataGrid' => 'git@github.com:spiral/data-grid-bridge.git',
-        'src/Bridge/Stempler' => 'git@github.com:spiral/stempler-bridge.git',
-        'src/Bridge/Monolog' => 'git@github.com:spiral/monolog-bridge.git',
-        'src/Bridge/Dotenv' => 'git@github.com:spiral/dotenv-bridge.git',
-
-        // Components
```

Comment:
I think there are no option to run it locally. Because the option `DIRECTORIES_TO_REPOSITORIES` removed since v9:
https://github.com/symplify/symplify/commit/7ecd39ea955bb91db15a20258333b2891003a903#diff-92b90ade6da2d2d856f195eeb4512a148e74cc32a107defefbc61f3c6a169a64L58

And replaced by a github action.